src/uproot/behaviors/TBranch.py

```diff
@@ -598,8 +598,8 @@ def dask(
     """
     files = uproot._util.regularize_files(files)
     library = uproot.interpretation.library._regularize_library(library)
-    if library.name != "np":
-        raise NotImplementedError()
+    # if library.name != "np":
+    #     raise NotImplementedError()
```

Comment:
In a committed PR, old code should be removed, not commented out.

But I think you still want to complain about Pandas.

```suggestion
    if library.name == "pd":
        raise NotImplementedError()
```